app/src/test/java/org/oppia/android/app/testing/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -101,82 +86,6 @@ class AdministratorControlsFragmentTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
-  @Test
-  fun testAdministratorControlsFragment_clickEditProfile_checkSendingTheCorrectIntent() {
-    launch<AdministratorControlsActivity>(
-      createAdministratorControlsActivityIntent(
-        0
-      )
-    ).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.edit_profiles_text_view))
-        .perform(click())
-      intended(hasComponent(ProfileListActivity::class.java.name))
-    }
-  }
-
-  @Test
-  fun testAdministratorControlsFragment_clickAppVersion_checkSendingTheCorrectIntent() {
-    launch<AdministratorControlsActivity>(
-      createAdministratorControlsActivityIntent(
-        0
-      )
-    ).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.administrator_controls_list)).perform(
-        scrollToPosition<RecyclerView.ViewHolder>(
-          3
-        )
-      )
-      onView(withId(R.id.app_version_text_view)).perform(click())
-      intended(hasComponent(AppVersionActivity::class.java.name))
-    }
-  }
-
-  @Test
-  @Config(qualifiers = "sw600dp")
-  fun testAdministratorControlsFragment_clickEditProfile_checkLoadingTheCorrectFragment() {
-    launch<AdministratorControlsActivity>(
-      createAdministratorControlsActivityIntent(
-        0
-      )
-    ).use {
-      testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.edit_profiles_text_view))
-        .perform(click())
-      it.onActivity { activity ->
-        val fragment =
-          activity.supportFragmentManager
-            .findFragmentById(R.id.administrator_controls_fragment_multipane_placeholder)
-        assertThat(fragment is ProfileListFragment).isTrue()
-      }
-    }
-  }
-
-  @Test
-  @Config(qualifiers = "sw600dp")
-  fun testAdministratorControlsFragment_clickAppVersion_checkLoadingTheCorrectFragment() {
```

Comment:
ditto

Reply:
This test is now handled both for mobile and tablet.